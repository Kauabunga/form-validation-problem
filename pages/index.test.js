import React from "react";
import { render } from "enzyme";

import Index from "./index";

it("should render the index and check it against a snapshot", () => {
  const component = render(<Index />);
  expect(component).toMatchSnapshot();
});
