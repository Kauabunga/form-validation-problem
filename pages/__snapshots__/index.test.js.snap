// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the index and check it against a snapshot 1`] = `
<form
  action="/api/awesome"
  method="post"
  novalidate=""
>
  <h1>
    Fill out this awesome form
  </h1>
  <fieldset>
    <h3>
      Your details
    </h3>
    <p
      class=""
    >
      <label
        class="label"
        for="email"
      >
        <span>
          Email
        </span>
        <input
          id="email"
          name="email"
          type="email"
          value=""
        />
      </label>
    </p>
    <p
      class=""
    >
      <label
        class="label"
        for="password"
      >
        <span>
          Password
        </span>
        <input
          id="password"
          name="password"
          type="password"
          value=""
        />
      </label>
    </p>
  </fieldset>
  <fieldset>
    <h3>
      Your animal
    </h3>
    <p
      class=""
    >
      <label
        class="label"
        for="colour"
      >
        <span>
          Colour
        </span>
        <select
          id="colour"
          name="colour"
        >
          <option
            selected=""
            value=""
          >
            Choose colour
          </option>
          <option
            value="blue"
          >
            Blue
          </option>
          <option
            value="green"
          >
            Green
          </option>
          <option
            value="red"
          >
            Red
          </option>
          <option
            value="black"
          >
            Black
          </option>
          <option
            value="brown"
          >
            Brown
          </option>
        </select>
      </label>
    </p>
    <p
      class=""
      id="animal"
    >
      <span
        class="label"
      >
        <span>
          Animal
        </span>
      </span>
      <label
        for="bear"
      >
        <input
          id="bear"
          name="bear"
          type="checkbox"
          value="bear"
        />
        <span>
          Bear
        </span>
      </label>
      <label
        for="tiger"
      >
        <input
          id="tiger"
          name="tiger"
          type="checkbox"
          value="tiger"
        />
        <span>
          Tiger
        </span>
      </label>
      <label
        for="snake"
      >
        <input
          id="snake"
          name="snake"
          type="checkbox"
          value="snake"
        />
        <span>
          Snake
        </span>
      </label>
      <label
        for="donkey"
      >
        <input
          id="donkey"
          name="donkey"
          type="checkbox"
          value="donkey"
        />
        <span>
          Donkey
        </span>
      </label>
    </p>
    <p
      class=""
    >
      <label
        class="label"
        for="tiger_type"
      >
        <span>
          Type of tiger
        </span>
        <input
          id="tiger_type"
          name="tiger_type"
          type="text"
          value=""
        />
      </label>
    </p>
  </fieldset>
  <fieldset>
    <p>
      <input
        type="submit"
        value="Create account"
      />
    </p>
  </fieldset>
</form>
`;
